doc/devices/braket_remote.rst

```diff
@@ -99,10 +99,49 @@ from :mod:`braket.pennylane_plugin.ops <.ops>`:
     braket.pennylane_plugin.GPi2
     braket.pennylane_plugin.MS
 
+Pulse Programming
+~~~~~~~~~~~~~~~~~
+
+The PennyLane-Braket plugin provides pulse-level control for the OQC Lucy QPU through PennyLane's `ParametrizedEvolution <https://docs.pennylane.ai/en/latest/code/api/pennylane.pulse.ParametrizedEvolution.html>`_
+operation. Compatible pulse Hamiltonians can be defined using the `qml.pulse.transmon_drive <https://docs.pennylane.ai/en/latest/code/api/pennylane.pulse.transmon_drive.html>`_
+function and used to create ``ParametrizedEvolution``'s using `qml.evolve <https://docs.pennylane.ai/en/stable/code/api/pennylane.evolve.html>`_:
+
+.. code-block:: python
+
+    duration = 15
+    def amp(p, t):
+        return qml.pulse.pwc(duration)(p, t)
+
+    dev = qml.device("braket.aws.qubit", wires=8, device_arn="arn:aws:braket:eu-west-2::device/qpu/oqc/Lucy")
+
+    drive = qml.pulse.transmon.transmon_drive(amplitude=amp, phase=0, freq=4.8, wires=[0])
+
+    @qml.qnode(dev)
+    def circuit(params, t):
+        qml.evolve(drive)(params, t)
+        return qml.expval(qml.PauliZ(wires=0))
+
+Note that the ``amplitude`` and ``freq`` arguments of ``qml.pulse.transmon_drive`` must be specified in :math:`\text{GHz}`. This will be internally
+converted into :math:`\text{rad/s}` for use with the Braket API. The ``phase`` must be specified in :math:`\text{radians}`.
+
+The pulse settings for the device can be obtained using the ``pulse_settings`` property. These settings can be used to describe the transmon
+interaction Hamiltonian using `qml.pulse.transmon_interaction <https://docs.pennylane.ai/en/latest/code/api/pennylane.pulse.transmon_interaction.html>`_:
+
+    .. code-block:: python
+
+        dev = qml.device("braket.aws.qubit", wires=8, device_arn="arn:aws:braket:eu-west-2::device/qpu/oqc/Lucy")
+        pulse_settings = dev.pulse_settings
+        H = qml.pulse.transmon_interaction(**pulse_settings, coupling=0.02)
+
+By passing ``pulse_settings`` from the remote device to ``qml.pulse.transmon_interaction``, an ``H`` Hamiltonian term is created using
+the constants specific to the hardware. This is relevant for simulating the hardware in PennyLane on the ``default.qubit`` device.
+
+Note that the user must supply coupling coefficients, as these are not available from the hardware backend.
```

Comment:
@Qottmann do you have a paper I can link to?

Reply:
I've updated this a bit (also the code block above it, the format was out of date). But upon further consideration, I wonder if we should just drop this here - we don't require creating an interaction term for the hardware upload. The `pulse_settings` functionality can be documented and commented on in the PennyLane demos where we simulate and compare, but it's not really needed for pulse upload to the hardware. So I think it might be confusing and unnecessary in this context; I think in the Braket documentation we should stick to how to interact with the hardware through the plugin.

Reply:
Not mentioning this here would surely be the safer side. As a user I'd still appreciate as much info about this as possible, so I am bit torn.

Here is [the paper](https://arxiv.org/pdf/1905.05670.pdf), see table 1, with the coupling being aroun 10 MHz

Reply:
The transition frequencies on their commercial device (lucy) are different, so I dont expect this to be accurate / up to date.